lua/entities/gmod_wire_hologram.lua

```diff
@@ -6,7 +6,7 @@ ENT.DisableDuplicator = true
 
 function ENT:SetPlayer(ply)
 	self:SetVar("Founder", ply)
-	self:SetVar("FounderIndex", ply:UniqueID())
+	self:SetVar("FounderIndex", ply:AccountID())
```

Comment:
Is this gonna be ok? I'm guessing we create this cause of it being in the `base_gmodentity`, however that uses `UniqueID`

https://github.com/Facepunch/garrysmod/blob/e7d41e003ad4b56b991406f821b164057f2985b6/garrysmod/gamemodes/sandbox/entities/entities/base_gmodentity.lua#L70-L81

Reply:
It will be OK, once Rubat approves my GMod Lua patch there... Then we just need to update advdupe2 here (since it uses `undo.GetTable()` which is indexed via `UniqueID`). Others will follow...